Source/Scene/SkyAtmosphere.js

```diff
@@ -76,6 +76,11 @@ define([
         this._command = new DrawCommand({
             owner : this
         });
+        this._defaultColor = {
+          red: 1,
+          green: 1,
+          blue: 1
+        };
```

Comment:
@emackey  thanks, I didn't know that.
Can you also tell me how to pass this color as an argument in the SkyAtmoshpere API.